File: Estrutura de Dados Simples/Matrizes_AeB.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] matrizA = new int[15];
        int[] matrizB = new int[15];

        System.out.println("---- Preenchendo a Matriz A ----");
        for (int i = 0; i < 15; i++) {
            while (true) {
                try {
                    System.out.print("Digite o " + (i + 1) + "º elemento para a matriz A: ");
                    matrizA[i] = Integer.parseInt(scanner.nextLine());
                    break;
                } catch (NumberFormatException e) {
                    System.out.println("Entrada inválida. Por favor, digite um número inteiro.");
                }
            }
        }

        for (int i = 0; i < 15; i++) {
            matrizB[i] = matrizA[i] * matrizA[i];
        }

            System.out.println("\n---- Matriz A ----");
        for (int elemento : matrizA) {
            System.out.print(elemento + " ");
        }

        System.out.println();
        System.out.println("\n---- Matriz B (Quadrados da Matriz A) ----");
        for (int elemento : matrizB) {
            System.out.print(elemento + " ");
        }

        System.out.println();
        scanner.close();
    }
}

File: Estrutura de Dados Simples/Matrizes_AeB.py
matriz_A = []
matriz_B= []

print("---Preenchendo a Matriz A ---")
for i in range(15):
    while True:
        try:
            elemento = int(input(f"Digite o {i+1}° elemento para a matriz A: "))
            matriz_A.append(elemento)
            break
        except ValueError:
            print("Entrada inválida. Por favor, digite um número inteiro.")
    
for elemento_a in matriz_A:
        matriz_B.append(elemento_a ** 2)

print("\n---Maytriz A ---")
print(matriz_A)

print("\n--- Matriz B (Quadrados da Matriz A) ---")
print(matriz_B)